crates/ra_syntax/tests/data/parser/inline/ok/0132_default_type.rs

```diff
@@ -0,0 +1 @@
+default type T = Foo;
```

Comment:
Rustc rejects this: https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=6b9352dc533fdd2ab96f3b3e6f0a38f3

Reply:
`default type` must be placed inside `impl` as:
```rust
impl Trait for Foo {
    default T = Debug;
}
```
cc https://rust-lang.github.io/rfcs/1210-impl-specialization.html#the-default-keyword

Reply:
I know, but here it isnt, yet ra seems to parse it fine.

Reply:
oops, I think I misunderstood something, fix soon.